### ISSUE TITLE
Members list is always empty

### ISSUE THREAD

**Issue by ichibod** [pusher/pusher-websocket-swift#133]
When following the example code below from the getting started:

```let pusher = Pusher(key: "YOUR_APP_KEY")

let chan = pusher.subscribeToPresenceChannel("presence-channel")

chan.bind(eventName: "pusher:subscription_succeeded", callback: { data in
    print("Subscribed!")
    print("I can now access myId: \(chan.myId)")
    print("And here are the channel members: \(chan.members)")
})
```

The channel's members is always empty. Regardless if there are actually members in there. Is there anything that could be done to work around this, or are we beholden to the added/removed events.

Thanks

**Comment by hamchapman** [comment]
I believe that the `members` property should be working as per the code in the README. 

I will be able to test this next week to see if it is indeed working as expected

**Comment by ichibod** [comment]
@hamchapman Any update on the tests?

**Comment by hamchapman** [comment]
Definitely a bug - thanks for reporting!

I've got a fix in a branch that I'm just about to push up to see if the tests all still pass on Travis.

If they do then I'll cut a new release this evening.

Thanks again!

**Comment by ichibod** [comment]
@hamchapman Happy to help! Thanks for the assistance!

**Comment by hamchapman** [comment]
Fixed in 4.0.2, which I've just released. Thanks!
